custom_components/tuya_local/devices/medion_life_p1002.yaml

```diff
@@ -0,0 +1,81 @@
+name: Portable Air Conditioner
+products:
+  - id: tqgodaa9suvtbflg
+    name: MEDION AIR Conditioner P1002
+primary_entity:
+  entity: climate
+  icon: "mdi:air-conditioner"
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cool
+              value: cool
+            - dps_val: Dyr
```

Comment:
Just to confirm, this typo is in the tuya implementation?

Reply:
yes